### ISSUE TITLE
cooroutine never awaited

### ISSUE THREAD

**Issue by bobloy** [Cog-Creators/Red-DiscordBot#1516]
### Type:

- [ ] Suggestion
- [x] Bug

### Brief description of the problem
Error message when removing instance in v3
### Expected behavior
Prompt to select instance to remove
### Actual behavior
![image](https://user-images.githubusercontent.com/955640/38520929-2f2f6ef0-3c12-11e8-949d-0aeb049fcf09.png)
```==========================
Red Discord Bot - Launcher
==========================

1. Run Red w/ autorestart in case of issues
2. Run Red
3. Update Red
4. Update Red (development version)
5. Create Instance
6. Remove Instance
7. Debug information (use this if having issues with the launcher or bot)
0. Exit
> 6
c:\program files\python35\lib\site-packages\redbot\launcher.py:322: RuntimeWarning: coroutine 'remove_instance' was never awaited
  remove_instance()
Press enter to continue.
```
### Steps to reproduce

1. Load v3
2. Attempt to remove instance

**Comment by laggron42** [comment]
also got that error, though it was only happening with me since I mess up a lot with the bot and python

**Comment by palmtree5** [comment]
Fixed in #1535
